### DISCUSSION TITLE
Add a disclaimer message in the footer of the chat UI

### DISCUSSION THREAD

**Discussion by tbendien** [bionic-gpt/bionic-gpt#544] [Ideas]
I'd like to add a disclaimer message under the prompt box that i can configure as the sysadmin such as:

"Always review content for accuracy and bias; use professional judgment and comply with school policies."

**Comment by tiswas247** [comment]
It's a good idea, we'll add that.

**Reply by tbendien** [reply]
any updates as to when this UX update will be added?

**Reply by tiswas247** [reply]
It's part of https://github.com/bionic-gpt/bionic-gpt/issues/554
